middleware/index.ts

```diff
@@ -221,14 +217,16 @@ export default function applySfFxMiddleware(request: any, state: any, resultArgs
 
     //use secret here in lieu of DEBUG runtime environment var until we have deployment time support of config var
     const debugSecret = getSecret("sf-debug", "DEBUG", logger);
-    logger.info(`DEBUG flag is ${debugSecret}`);
     if(debugSecret || LoggerLevel.DEBUG === logger.getLevel() || process.env.DEBUG) {
+        logger.info(`DEBUG flag is ${debugSecret}`);
```

Comment:
@dhagberg-sf , this needs to be outside of the if, if it's already inside if, I already know it's set

the following is what I have

    `logger.info(`DEBUG flag is ${debugSecret ? debugSecret : 'unset'}`);`

Reply:
fixed.